sites/vehicleservicepros.com/server/templates/website-section/blogs.marko

```diff
@@ -0,0 +1,160 @@
+    </marko-web-page-wrapper>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">Latest Blogs</h1>
+        <marko-web-query|{ nodes }|
+          name="all-published-content"
+          params={ contentTypes: ["Blog"], limit: 5, queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes with-teaser=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">IN THE DRIVER'S SEAT</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={
+            limit: 3,
+            sectionAlias: "blogs/drivers-seat",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">KOLMAN'S KOMMENTS</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content",
+          params={
+            limit: 3,
+            sectionAlias: "blogs/kolmans-komments",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">GUEST BLOGS</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content",
+          params={
+            limit: 3,
+            sectionAlias: "blogs/guest-blogs",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">INDUSTRY INSIGHTS</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={
+            limit: 3,
+            sectionAlias: "blogs/industry-insights",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">LABOR TIME</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={
+            limit: 3,
+            sectionAlias: "shop-operations/employees-hiring",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+  </@page>
+  <@below-page>
+    <marko-web-load-more
+      component-name="content-card-deck-flow"
+      fragment-name="content-list"
+      query-name="all-published-content"
+      query-params={ contentTypes: ["Video"], limit: 10, skip: 5 }
```

Comment:
the load more is displaying published video content... this shouldn't be the case. Should this page event have a load more block?